aspnet-core/src/AbpCompanyName.AbpProjectName.Core/Authorization/Users/UserManager.cs

```diff
@@ -54,5 +56,32 @@ public class UserManager : AbpUserManager<Role, User>
                 settingManager)
         {
         }
+
+        public virtual async Task<bool> VerifyPasswordResetCodeAsync(User user, string passwordResetCode)
+        {
+            return user.PasswordResetCode == passwordResetCode;
+        }
+
+        public virtual async Task<IdentityResult> BasicResetPasswordAsync(User user, string passwordResetCode, string newPassword)
+        {
+            ThrowIfDisposed();
```

Comment:
Why you needed this line?

Reply:
@alirizaadiyahsi Identity's UserManager methods always make such check. [Example](https://github.com/aspnet/AspNetIdentity/blob/master/src/Microsoft.AspNet.Identity.Core/UserManager.cs#L502). I can remove it if you want.